container.go

```diff
@@ -29,6 +29,7 @@ type ContainerProvider interface {
 	CreateContainer(context.Context, ContainerRequest) (Container, error) // create a container without starting it
 	RunContainer(context.Context, ContainerRequest) (Container, error)    // create a container and start it
 	Health(context.Context) error
+	Config() TestContainersConfig
```

Comment:
@baez90 please take a look at this for the Podman support 👀

Reply:
I'm sorry for that @baez90. Long-running branches have this kind of complexities. I truly appreciate your effort in the Podman support, and I think it's close to be there 🙏 

![image](https://media.giphy.com/media/42YlR8u9gV5Cw/giphy.gif)